File: Programming Basics with Python/T04 - Conditional Statements (Advanced)/exercise/01. Cinema.py
screening_type = str(input())
rows = int(input())
columns = int(input())
capacity = rows * columns
profit = 0

if screening_type == "Premiere":
    profit = capacity * 12
    print(f"{profit:.2f} leva")

elif screening_type == "Normal":
    profit = capacity * 7.50
    print(f"{profit:.2f} leva")

elif screening_type == "Discount":
    profit = capacity * 5
    print(f"{profit:.2f} leva")

File: Programming Basics with Go/T04 - Conditional Statements (Advanced)/exercise/01. Cinema.go
package main

import "fmt"

func main () {
	//бр. билети = бр. места = редове * колони
	//цена за един билет -> зависи от вид на прожекцията
	//печалба = бр. билети * цена за един билет

	var typeProjection string
	fmt.Scanln(&typeProjection)

	var rows, cols int
	fmt.Scanln(&rows)
	fmt.Scanln(&cols)

	var countTickets int = rows * cols
	var pricePerTicket float32 = 0

	//•	Premiere – премиерна прожекция, на цена 12.00 лева.
	//•	Normal – стандартна прожекция, на цена 7.50 лева.
	//•	Discount – прожекция за деца, ученици и студенти на намалена цена от 5.00 лева.

	switch typeProjection {
		case "Premiere":
			pricePerTicket = 12
		case "Normal":
			pricePerTicket = 7.50
		case "Discount":
			pricePerTicket = 5.00
	}

	var profit float32 = float32(countTickets) * pricePerTicket
	fmt.Printf("%.2f leva", profit)


}